### ISSUE TITLE
Clarification needed on target-settings

### ISSUE THREAD

**Issue by nigelhaslam** [aktos-io/dcs-tools#12]
Please excuse such simple questions but when I run the command:
**./make-target-settings** what does the 'target' refer to.. the destination machine where the backup is to be copied to, or the raspberry pi which I'm trying to back up?

If you could add step by step instructions, with details of which machine the command is being performed on, that would help.  I have managed to set up a Putty ssh connection from my workstation to the Raspberry Pi but I have been getting confused whether it makes any difference when I'm configuring the target and setup files.
thanks

**Comment by ceremcem** [comment]
Main confusion stays here: The machine you use for backups and issuing commands (such as `make ssh`) is your current machine (for example, your laptop). 

History of this tool might help a bit: We have our laptops and we need to make/change programs on the controllers in the field which have Linux on them. We might mess up whole application of the **target**, so we should backup (to our laptop) before doing any operations. We don't want to remember the **target** controller's IP address and password every time. So this tool is created.

Well, the last clue: Are you using Windows? :)

**Comment by nigelhaslam** [comment]
OK

The history does help.  The 'target' for you, is obviously the machine you
are going to work on (and therefore need a backup of).

I'm mostly a windows guy but I used to use Unix 20 years ago, so some of
the everyday commands are familiar but installations are out of my comfort
zone.

I really like your software, I hope I don't wear you down before I can
figure it out.

Cheers
Nigel

*Nigel Haslam* *| Director *

*Motion Circus Pty Ltd. Byron Bay*
*t**: + 61 2 8007 7338 **|**m**: * *+ 61  403 020 126*
*w: www.motioncircus.com <http://www.motioncircus.com/>*
link to website <http://motioncircus.com>

**Comment by nigelhaslam** [comment]
Sorry,

to be clear.  My workstation is a Win10 machine.  My Raspberry is running
Raspbian.

I have been using the GitBash shell on my workstation, which seems to give
me a linux like environment for ssh to Raspberry

N

*Nigel Haslam* *| Director *

*Motion Circus Pty Ltd. Byron Bay*
*t**: + 61 2 8007 7338 **|**m**: * *+ 61  403 020 126*
*w: www.motioncircus.com <http://www.motioncircus.com/>*
link to website <http://motioncircus.com>

**Comment by ceremcem** [comment]
No problem for me, but keep in mind that this tool is intended to be used on a Linux machine where `make` command is present by default and is never tested on a Windows machine.

**Comment by nigelhaslam** [comment]
Thanks for the heads up.. I will look into the make command, which is new
to me, and check if my b
Git Bash shell is fully compatible with it.  I know some Linux emulators
fall short on windows.

Will let you know what I find.

Nigel

On 25 Oct 2017 17:38, "Cerem Cem ASLAN" <notifications@github.com> wrote:

No problem for me, but keep in mind that this tool is intended to be used
on a Linux machine where make command is present by default and is never
tested on a Windows machine.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/aktos-io/dcs-tools/issues/12#issuecomment-339229790>,
or mute the thread

**Comment by nigelhaslam** [comment]
Hi,

it turns out my Git Bash windows installation was lacking the
'make' command  but I found a way to install it on this page

https://gist.github.com/evanwill/0207876c3243bbb6863e65ec5dc3f058



*Nigel Haslam* *| Director *

*Motion Circus Pty Ltd. Byron Bay*
*t**: + 61 2 8007 7338 **|**m**: * *+ 61  403 020 126*
*w: www.motioncircus.com <http://www.motioncircus.com/>*
link to website <http://motioncircus.com>

**Comment by ceremcem** [comment]
Anything left with the issue to address to?

**Comment by nigelhaslam** [comment]
Hey Cerem,
I hope you had a nice weekend.
I have decided it would be useful if I had a fully linux environment to work in, so I've ordered another RaspberryPi.  The Zero model this time, which I hope will work as my time lapse rig controller.  It only draws half the power of the bigger models, so I'm it should run for twice as long on the battery supply.  It should arrive very soon, at which point, I will see if I can get your tools working with it.

In the meantime, if you just add a disclaimer to your GitHub usage docs, letting people know they need a full linux environment, I think that would be a good addition.
Cheers
Nigel
